docs/usage/guides/plugin-builder.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Write Tauri Plugins
+---
+
+import Alert from '@theme/Alert'
+
+<Alert title="Note" icon="info-alt">
+This guide uses the `plugin::Builder` introduced in `v1`, if you're looking for the old version using trait implementations [see here](./plugin.md)
```

Comment:
Can't we combine both? maybe improve the old guide and add it under **Advanced** section or something.

Reply:
yeah you're right.

Reply:
fixed!